src/main/java/uk/gov/companieshouse/exemptions/service/ExemptionsServiceImpl.java

```diff
@@ -52,16 +52,22 @@ public ServiceStatus upsertCompanyExemptions(String contextId, String companyNum
                         .ifPresentOrElse(document::setCreated,
                                 () -> document.setCreated(new Created().setAt(document.getUpdated().at())));
 
+                repository.save(document);
+                logger.info(String.format("Company exemptions for company number: %s updated in MongoDb for context id: %s",
+                        companyNumber,
+                        contextId));
+
                 ServiceStatus serviceStatus = exemptionsApiService.invokeChsKafkaApi(new ResourceChangedRequest(contextId, companyNumber, null, false));
-                logger.info(String.format("ChsKafka api CHANGED invoked updated successfully for context id: %s and company number: %s",
+                logger.info(String.format("ChsKafka api CHANGED invoked for context id: %s and company number: %s",
```

Comment:
Should probably get rid of this log message, we log on 66 or 68 anyway.

Reply:
Sorted